app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe | str) -> None | str:
```

Comment:
Code Style: It's a good practice to use type annotation for your function parameters and return type. However, you should avoid using the `None` type as a return type. A function should always return a value. If the function doesn't need to return a value, you should not include `None` in the return type annotation.